README.md

```diff
@@ -166,6 +166,13 @@ The `--prebuiltDirectory` flag is useful for working with Webpack for example. I
 ## Handling `npm link` and Dependencies With Local Paths
 Perhaps the easiest way to handle these cases is to bundle the code using Webpack and use the `--prebuiltDirectory` flag to package the output for deployment.
 
+## ScheduleEvents
+#### Optional Parameter
+When using the eventSourceFile flag (-S or --eventSourceFile) to set a ScheduleEvent trigger, you can pass an optional _ScheduleDescription_ key into the ScheduleEvent object with a custom description for the CloudWatch event rule you are defining. By default, node-lambda generates a _ScheduleDescription_ for you based on the ScheduleName and ScheduleExpression of the rule.
+
+#### Note on ScheduleState for ScheduleEvents
```

Comment:
@DeviaVir I added this subsection to the ScheduleEvents section because I feel this behavior deserves to be noted as it may be confusing to someone trying to deploy a Lambda with a disabled trigger. I wanted to bring it to your attention because I'm not certain if this is the intended behavior, but I can confirm that this is, in fact, how it currently works.